core/templates/dev/head/pages/question_player/question_player_directive.html

```diff
@@ -5,6 +5,11 @@ <h3>Question <[currentQuestion]>/<[totalQuestions]></h3>
       <div class="current-progress" style="width: <[currentProgress]>%"></div>
     </div>
   </div>
+  <div class="oppia-question-player-body">
+    <conversation-skin question-player-config="questionPlayerConfig"></conversation-skin>
+    <attribution-guide></attribution-guide>
+  </div>
+  <exploration-footer></exploration-footer>
```

Comment:
Can we have a generic name for this directive?

Reply:
Maybe `player-footer`?

Reply:
So it actually is part of the exploration player, it shows the hints and solutions of the exploration. Is player too generic?